content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/configuration.md

```diff
@@ -16,18 +16,56 @@ Since both O3DE Gems and projects use the same CMake build functions, these inst
 
 ## Build setup
 
-The first step of enabling the Multiplayer Gem for a project is to make sure that the `<ProjectName>.Static` target of your project includes the correct dependencies. Make the following edits to the `<ProjectName>.Static` target:
+As usual, the first step of enabling the Multiplayer Gem is to make sure the gem is [added to the project](/docs/user-guide/project-config/add-remove-gems/).
```

Comment:
"As usual" is not adding anything here. Keep it simple, "Start of by ensuring that Multiplayer Gem is configured for the project... ] or similar.

Reply:
Good call! K.I.S.